SU-SWS/stanford_profile_helper
# D8CORE-2899: removing the character limits on the wysiwyg text field

# READY FOR REVIEW
- This is for 1.5.1. It was posted early so @katrialesser can review for impact in her projects.

# Summary
- Removing the character limits on the wysiwyg text field. We want them to widen up to 100 

# Needed By (Date)
- For 1.5.1. 
- Needs review from the other subtheme teams.

# Urgency
- High

# Steps to Test

1. Pull in this branch and the one on [Stanford Basic](https://github.com/SU-SWS/stanford_basic/compare/D8CORE-2899-v2?expand=1).
2. Refresh everything.
3. Make new text areas and add content with the wysiwyg editor. Verify the character limit is 100.
4. Test on several different page types.

# Affected Projects or Products
- Basic, Adapt, SOE

# Associated Issues and/or People
- D8CORE-2899
- https://github.com/SU-SWS/stanford_basic/compare/D8CORE-2899-v2?expand=1
- The Jira ticket has all the classes changed listed out.
- @katrialesser @rmundstock @cjwest 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
